aptos-move/framework/src/natives/cryptography/algebra/mod.rs

```diff
@@ -0,0 +1,319 @@
+// Copyright © Aptos Foundation
+
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{
+    natives::{
+        cryptography::algebra::gas::GasParameters,
+        helpers::{make_safe_native, SafeNativeContext, SafeNativeError, SafeNativeResult},
+    },
+    safely_pop_arg,
+};
+use aptos_types::on_chain_config::{FeatureFlag, Features, TimedFeatures};
+use ark_serialize::{CanonicalDeserialize, CanonicalSerialize};
+use better_any::{Tid, TidAble};
+use move_core_types::language_storage::TypeTag;
+use move_vm_runtime::native_functions::NativeFunction;
+use move_vm_types::{
+    loaded_data::runtime_types::Type,
+    values::{Value, VectorRef},
+};
+use smallvec::{smallvec, SmallVec};
+use std::{any::Any, collections::VecDeque, hash::Hash, ops::Add, rc::Rc, sync::Arc};
+
+pub mod gas;
+
+/// Equivalent to `std::error::not_implemented(0)` in Move.
+const MOVE_ABORT_CODE_NOT_IMPLEMENTED: u64 = 0x0C0000;
```

Comment:
For a bit more clarity, can you write it as `0x0C_0000`?

Reply:
Fixed.